### PR TITLE
Sensitive Logs Data

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Wholesale edit of this docs, specifically editing to update the query example per request

### Motivation
Jira request

### Preview
https://docs-staging.datadoghq.com/sarina/sensitive-logs-data/logs/guide/control-sensitive-logs-data

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by jtappa** [comment]
The build for this failed with:
`Error: Error building site: "/go/src/github.com/DataDog/documentation/content/en/logs/guide/control-sensitive-logs-data.md:65:121": unrecognized character in shortcode action: U+003E '>'. Note: Parameters with non-alphanumeric args must be quoted`

File: content/en/logs/guide/control-sensitive-logs-data.md [hunk 1]
```diff
@@ -16,100 +16,94 @@ further_reading:
...
 This is an optional step to prevent additional sensitive data from being sent to Datadog. However keep in mind that you’ll still need to address the sensitive data already sent to Datadog.
 
-* Find which index(es) logs holding sensitive data are susceptible to be routed.
-* For each index, add an exclusion filter based on the sensitive outline query. It has to be **in the first place**. 
-
-{{< img src="logs/guide/sensitive/sensitive_exclusion-filters.png" alt="Sensitive Exclusion Filters" style="width:80%;" >}}
+* Find which index(es) that hold sensitive data are susceptible to routing.
+* For each index, add an exclusion filter based on the sensitive outline query.
 
+{{< img src="logs/guide/sensitive/sensitive_exclusion-filters.png" alt="Sensitive Exclusion Filters" style="width:80%;" >}
```
Comment (jtappa) [inline content/en/logs/guide/control-sensitive-logs-data.md:65]: I think you might be missing a slash or something in this tag and that's what's making it fail?

Comment (jtappa) [inline content/en/logs/guide/control-sensitive-logs-data.md:65]:
```suggestion
{{< img src="logs/guide/sensitive/sensitive_exclusion-filters.png" alt="Sensitive Exclusion Filters" style="width:80%;" >}}
```

Comment (jtappa) [reply content/en/logs/guide/control-sensitive-logs-data.md]: Got it, was a missing `}`

File: content/en/logs/guide/control-sensitive-logs-data.md [hunk 2]
```diff
@@ -16,100 +16,94 @@ further_reading:
...
 
-- If you have intentionally set up your logs to have sensitive data for legitimate troubleshooting and auditing purposes, use Roles Based Access Control to ensure that you have set up appropriate restrictions to ensure that only entitled users who have access to your Datadog account can access this data. For more information, see the [Logs Roles Based Access Control (RBAC) User's Guide][1] and learn how to configure it for your organization.
-- Address any unintentional logging of sensitive data to preemptively address any concerns down the road. Continue with this guide to learn about addressing that concern.
+- If you have intentionally set up your logs to have sensitive data for legitimate troubleshooting and auditing purposes, use the **Roles Based Access Control** to ensure that you have set up appropriate restrictions so only entitled users who have access to your Datadog account can access this data. For more information, see the [Logs Roles Based Access Control (RBAC) User's Guide][1] to learn how to configure it for your organization.
```
Comment (jtappa) [inline content/en/logs/guide/control-sensitive-logs-data.md:21]: I don't know why but "entitled" bristles me. 😂  Could we change it to "authorized" maybe?

Comment (sarina-dd) [reply content/en/logs/guide/control-sensitive-logs-data.md:21]: Definitely, this one slipped by! Will update.

Comment (sarina-dd) [inline content/en/logs/guide/control-sensitive-logs-data.md:21]:
```suggestion
- If you have intentionally set up your logs to have sensitive data for legitimate troubleshooting and auditing purposes, use the **Roles Based Access Control** to ensure that you have set up appropriate restrictions so only authorized users who have access to your Datadog account can access this data. For more information, see the [Logs Roles Based Access Control (RBAC) User's Guide][1] to learn how to configure it for your organization.
```

**Review by jtappa — APPROVED** [review]

**Pull request merged**